backend/src/db.ts

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable no-console */
+import { Pool } from 'pg';
+
+const pool = new Pool({
+  user: 'fluxstride',
```

Comment:
You shouldn't expose these database credentials, they should be saved in your .env.

Reply:
```
const pool = new Pool({
  user: process.env.DB_USER,
  host: process.env.DB_HOST,
  ...
 })
  ```